jsonargparse/actions.py

```diff
@@ -70,8 +70,11 @@ def _find_action_and_subcommand(
     if exclude is not None:
         actions = [a for a in actions if not isinstance(a, exclude)]
     fallback_action = None
+
+    ALLOW_ALIAS = True
+
     for action in actions:
-        if action.dest == dest:
+        if action.dest == dest or (ALLOW_ALIAS and dest in get_alias_dest(action)):
```

Comment:
Oh, right. Only the actions defined in jsonargparse inherit from this class. However, there are normal argparse actions and `_StoreAction` is one of them. Patching the argparse Action class to have this property would not be nice, but can be an option. Another option is to make it a normal attribute instead of a property and have a function (say `add_aliases(action)`) to add it. This function would be called for example before this line
https://github.com/omni-us/jsonargparse/blob/927ec027f41520cc768d501387ea678a096848c8/jsonargparse/core.py#L134

Reply:
Patching of the `argparse.Action` class seems a bit heavy handed and prone to breakage depending on import and instance creation order. 

Patching an aliases attribute in on an instance level in the place you mentioned seems more reasonable, but I'm wondering if it makes the code harder to read. For example, a reader sees this magic alias property used, but doesn't see where it comes from. On the other hand, if there is a `get_aliases` function (which checks for and then sets the attribute so it isn't recomputed), then it's clear where the attribute is coming from. Either of the two options will work though, so let me know if you have a preference.